src/gpio.rs

```diff
@@ -9,6 +9,32 @@ pub trait GlbExt {
     fn split(self) -> Parts;
 }
 
+#[allow(non_camel_case_types)]
```

Comment:
is that really needed - looks camel case to me?

Reply:
No, it is a left over before I renamed it. I will remove that.